examples/ticks/index.js

```diff
@@ -1,25 +1,34 @@
-import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api";
+import DerivAPIBasic from "https://cdn.skypack.dev/@deriv/deriv-api/dist/DerivAPIBasic";
+
 const app_id = 1089; // Replace with your app_id or leave as 1089 for testing.
 const connection = new WebSocket(
   `wss://ws.binaryws.com/websockets/v3?app_id=${app_id}`
 );
 const api = new DerivAPIBasic({ connection });
+const tick_stream = api.subscribe({ ticks: "R_100" });
 
-const tickResponse = (res) => {
+const tickResponse = async (res) => {
   const data = JSON.parse(res.data);
-  console.log(data.tick);
-
-  // After 15 seconds, will stop tick response.
-  setTimeout(async () => {
+  if (data.error !== undefined) {
+    console.log("Error : %o", data.error.message);
```

Comment:
Is that ok to have console.log here? we should removed all of them when we are going to create a PR.

Reply:
Hi Niloo, thanks for the question. Yes, the console logs in the code are intended, it's nice to use in codesandbox, as it shows directly to the user what type of data the API returns which is the goal of these code examples. Then it is up to that developer him-/herself to determine what to do with it.